src/geo/leaflet/leaflet-tiled-layer-view.js

```diff
@@ -49,12 +37,34 @@ LeafletTiledLayerView.prototype = _.extend(
     },
 
     _modelUpdated: function () {
-      L.Util.setOptions(this.leafletLayer, generateLeafletLayerOptions(this.model));
-      this.leafletLayer.setUrl(this.model.get('urlTemplate'));
+      L.Util.setOptions(this.leafletLayer, _generateLeafletLayerOptions.call(this, this.model));
+      this.leafletLayer.setUrl(_getUrlTemplate(this._isHdpi(), this.model));
+    },
+
+    _isHdpi: function () {
+      return window && window.devicePixelRatio && window.devicePixelRatio > 1;
     }
   }
 );
 
 LeafletTiledLayerView.prototype.constructor = LeafletTiledLayerView;
 
+var _generateLeafletLayerOptions = function (layerModel) {
```

Comment:
again just a matter of personal taste:
I prefer to create a function using [function declarations](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/function) instead [function expressions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/function)

nowadays it doesn't matter but a few years ago with function expressions you assign an anonymous function to a variable so the stack traces were all `anonymous` functions